sopel/modules/reddit.py

```diff
@@ -188,15 +193,56 @@ def comment_info(bot, trigger, match):
     bot.say(message)
 
 
-# If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, commanded=False):
+    """Shows information about the given subreddit"""
+    r = bot.memory['reddit_praw']
+    try:
+        r.subreddits.search_by_name(match, exact=True)
+    except prawcore.exceptions.NotFound:
+        if commanded:
+            bot.say('No such subreddit.')
+        # Fail silently if it wasn't an explicit command.
+        return NOLIMIT
+
+    try:
+        s = r.subreddit(match)
+        s.subreddit_type
+    except prawcore.exceptions.Forbidden:
+        bot.say("r/" + match + " appears to be a private subreddit!")
+        return NOLIMIT
+    except prawcore.exceptions.NotFound:
+        bot.say("r/" + match + " appears to be a banned subreddit!")
+        return NOLIMIT
+
+    link = "https://www.reddit.com/r/" + s.display_name
```

Comment:
Might as well save a few characters, and be consistent with the Redditor function.

```suggestion
    link = "https://reddit.com/r/" + s.display_name
```

(Alternatively, update the Redditor function to include `www.`